Standards/scs-xxxx-v1-networking-standard.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Networking Standard
+type: Standard
+status: Proposal
+track: IaaS
+---
+
+## Introduction
+
+[...]
+
+## Motivation
+
+[...]
+
+## Design Considerations
+
+Networking setups can vary between OpenStack clouds in a number of aspects. The focus of this standard are those aspects that have an impact on the cloud user, which can be broadly grouped into two areas:
+* API behavior, which includes policy rules and the availability of API extensions,
+* and CSP-managed resources, which includes provider networks and their subnets, as well as floating IPs and subnet pools.
+
+### Options considered
+
+#### API Extensions
+
+The OpenStack Networking API is more modular than many other OpenStack APIs, adding new functionality as optional API extensions, rather than a linear sequence of micro-versions. Which of those extensions are available from Neutron depends on the enabled plugins and back-end drivers.
+
+Mandating a minimum set of enabled extensions will ensure that users have a consistent feature-set available across SCS clouds, but should not unnecessarily restrict the choice of back-end drivers for CSPs.
+
+[...]
+
+#### Policies
+
+[...]
```

Comment:
One approach might be to mandate policies to be equal or stricter than the neutron default policies, (with possible exemptions for CSP-internal projects/users). Is something like this already part of the standards? should it be?